app/lab/package.json

```diff
@@ -85,7 +85,7 @@
     "@jupyterlite/types": "~0.1.0-beta.11",
     "@jupyterlite/ui-components": "~0.1.0-beta.11",
     "@lumino/algorithm": "~1.9.1",
-    "@lumino/application": "~1.28.1",
+    "@lumino/application": "^1.27.0",
```

Comment:
Wondering if this one needs the `^`? Or should all  `@lumino` dependencies use the same `~` for consistency?

Reply:
was emulating [upstream](https://github.com/jupyterlab/jupyterlab/blob/v3.4.4/jupyterlab/staging/package.json#L113). with the `~` was getting some `console.warning`, but maybe adding it to the `resolution` in the root `package.json` will resolve it...

Reply:
Alright let's keep that way then :+1: